platform_channels/lib/src/pet_list_screen.dart

```diff
@@ -25,7 +25,7 @@ class _PetListScreenState extends State<PetListScreen> {
     BasicMessageChannel('stringCodecDemo', StringCodec())
         .setMessageHandler((message) async {
       if (message == null) {
-        scaffoldKey.currentState.showSnackBar(
+        scaffoldMessengerKey.currentState.showSnackBar(
```

Comment:
Since, we are migrating to `ScaffoldMessenger`, we will no longer need key to access the currrentState using a key. You can simply use `ScaffoldMessenger.of(context)` to access the `ScaffoldMessengerState`. @RedBrogdon will be happy to see Key removed :)

Also, JFYI, this will given an error about currentState being null. In order to make it work, you need to wrap `Scaffold` with `ScaffoldMessenger` and pass the scaffoldMessengerKey in `ScaffoldMessenger.key`.

Refer to the [migration guide](https://flutter.dev/docs/release/breaking-changes/scaffold-messenger#migration-guide) for more information.

Reply:
Thank you so much @AyushBherwani1998  for guiding me  and reviewing the PR  I'll do the changes and update the PR 
😊.